articles/quickstart/native/android/07-authorization.md

```diff
@@ -18,28 +18,34 @@ This step demonstrates how to use Auth0 to create access roles for your users. W
   ]
 }) %>__
 
-## Before Starting
+## Before You Start
 
-Be sure that you have completed the [user profile](04-user-profile) quickstart.
+::: note
+Before you continue with this tutorial, make sure that you have completed the [User Profile](04-user-profile) tutorial.
+:::
+
+## Create a Rule to Assign Roles
 
-## Create A Rule To Assign Roles
+Create a rule that assigns the following access roles to your user: 
+* An admin role
+* A regular user role
 
-First, you need to create a rule that assigns your users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](${manage_url}/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
+To assign roles, go to the [New rule](${manage_url}/#/rules/new) page. In the **Access Control** section,  select the **Set roles to a user** template. 
+
+Edit the following line from the default script to match the conditions that fit your needs:
 
 ```java
 if (user.email.indexOf('@example.com') > -1)
 ```
-to match the condition that fits your needs.
 
-By default, it says that if the user email contains `@example.com` then they will be given an `admin` role, otherwise a regular `user` role.
+The default rules for assigning access roles are:
+* If the user's email contains @example.com, the user gets the admin roles anything else, the user gets the regular user role
```

Comment:
If the user's email contains `@example.com`, the user gets the admin roles. Anything else, the user gets the regular user role.